### PR TITLE
Enable pylint for all python tests

### PR DESCRIPTION
**Pull request by chhetripradeep**
### Description

- As part of https://github.com/ClickHouse/ClickHouse/pull/49869 we noticed that we already use pylint, just that it wasn't enabled for integration tests.
- Adding myself to trusted contributor.

### Changelog category (leave one):

- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Comment by robot-ch-test-poll3** [comment]
This is an automated comment for commit cfe39b7375b27fbfe939866c99600eb9198caa47 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/50174/cfe39b7375b27fbfe939866c99600eb9198caa47/ci_running.html)
The overall status of the commit is 🟡 pending

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🟡 pending</td></tr>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>🟢 success</td></tr>
</table>

**Comment by Felixoid** [comment]
Looking at the output, I'd suggest disabling some particular lines with `import-error` and comment, that it's for style-check only

**Comment by qoega** [comment]
I'm also not sure about 
>/ClickHouse/tests/sqllogic/test_runner.py:300:8: W1203: Use lazy % formatting in logging functions (logging-fstring-interpolation)

It is nice to use fstring everywhere if there is no critical problem with it

**Comment by Felixoid** [comment]
No, this is intended and very useful https://pylint.pycqa.org/en/latest/user_guide/messages/warning/logging-fstring-interpolation.html

File: tests/integration/ci-runner.py [hunk 1]
```diff
@@ -314,13 +319,14 @@ def _pre_pull_images(self, repo_path):
                 )
                 return
             except subprocess.CalledProcessError as err:
-                logging.info("docker-compose pull failed: " + str(err))
+                logging.info("docker-compose pull failed: %s", str(err))
```
Comment (Felixoid) [inline tests/integration/ci-runner.py:322]: `str(err)` here and everywhere else is redundant. Each parameter in `logging.info` and similar methods are translated into `str` explicitly later. So we can save some unnecessary actions by passing just `err`, without converting into strings

File: tests/integration/conftest.py [hunk 2]
```diff
@@ -1,7 +1,9 @@
-from helpers.cluster import run_and_check
+#!/usr/bin/env python3
+
 import pytest
 import logging
 import os
+from helpers.cluster import run_and_check
```
Comment (Felixoid) [inline tests/integration/conftest.py:6]: very minor thing, but the empty line between stdlib and local imports improves readability

**Comment by alexey-milovidov** [comment]
Let's continue.

**Comment by Felixoid** [comment]
We should also switch to https://github.com/pylint-dev/pylint/releases/tag/v3.0.0b0 which contains https://github.com/pylint-dev/pylint/pull/7163

So we can repository-wide config that will be respected by all IDE

**Comment by Felixoid** [comment]
https://github.com/ClickHouse/ClickHouse/blob/master/docker/test/style/Dockerfile#L22

The [3.0.0](https://github.com/pylint-dev/pylint/releases/tag/v3.0.0) is just out.

**Comment by Felixoid** [comment]
@chhetripradeep I continue your work in #60408

**Pull request closed without merging**